README.md

```diff
@@ -3,28 +3,72 @@ A modified version of the [Synthetix](https://github.com/Synthetixio/synthetix)
 
 ## Overview
 
-The [`StakingRewards`](contracts/StakingRewards.sol) contract in this repository is a flattened and modified version of the SNX [staking rewards](https://github.com/Synthetixio/synthetix/blob/master/contracts/StakingRewards.sol) contract.
+The [`MultiRewards`](contracts/MultiRewards.sol) contract in this repository enables distribution of multiple reward tokens for staked users.  It is a flattened and modified version of the SNX [staking rewards](https://github.com/Synthetixio/synthetix/blob/master/contracts/StakingRewards.sol) contract.
 
-TODO
+## How it Works
+
+As an example, assume we desire users to stake an ERC20 Token `Base Token` ($BASE).  As a reward for staking their token, we may wish to offer users multiple different tokens.  For the sake of this example, we'll assume we have two different amount of governance tokens, `Reward Token 1` ($ONE) and `Reward Token 2` ($TWO), which we want to release over different schedules.
+
+ * We begin by deploying the MultiReward contract with a link to $BASE as our staking token.
+ * For both token $ONE and $TWO the contract's _Owner_ calls *addReward*
```

Comment:
```suggestion
 * For both token $ONE and $TWO the contract's _Owner_ calls `addReward`
```

Reply:
Code snippets should always be in backticks.  Please apply this to all the spots where you're currently using bold (asterisks)